src/Browser.php

```diff
@@ -249,6 +249,21 @@ public function move($x, $y)
         return $this;
     }
 
+    /**
+     * Scroll screen to element at the given selector.
+     *
+     * @param  string  $selector
+     * @return $this
+     */
+    public function scrollTo($selector)
+    {
+        $selector = $this->resolver->format($selector);
+
+        $this->driver->executeScript("$(\"html, body\").animate({scrollTop: $(\"$selector\").offset().top}, 0);");
```

Comment:
Does `jQuery` or `$` always present?

Reply:
@crynobone hmm good remark I'll send in a PR for this.

Reply:
https://github.com/laravel/dusk/pull/686